config/Dockerfiles/Jenkinsfile

```diff
@@ -4,8 +4,7 @@ env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/linchpin'
 env.ghprbActualCommit = env.ghprbActualCommit ?: 'develop'
 env.ghprbSourceBranch = env.ghprbSourceBranch ?: 'develop'
 env.ghprbTargetBranch = env.ghprbTargetBranch ?: 'develop'
-env.ghprbPullAuthorLogin = env.ghprbPullAuthorLogin ?: ''
-env.ghprbPullId = env.ghprbPullId ?: ''
+env.ghprbPullAuthorLogin = env.ghprbPullAuthorLogin ?: '' env.ghprbPullId = env.ghprbPullId ?: ''
```

Comment:
this shouldn't all be on one line should it?

Reply:
That shouldn't have been changed at all, thanks for the catch